envs/tensorflow-env.yaml

```diff
@@ -26,7 +26,7 @@ packages:
   - feedstock : tensorflow-text
   - feedstock : tensorflow-model-optimization
   - feedstock : typeguard
-  - feedstock : tensorflow-addons
+  - feedstock : tensorflow-addons                   #[build_type != 'nnpa']
   - feedstock : tensorflow-io-gcs-filesystem # [not s390x]
```

Comment:
Let's not do this. Instead we will add a fix in tensorflow-addons-feedstock to make the recipe depend on tensorflow-base instead of individual variants.

Reply:
ok

Reply:
@npanpaliya Sure, Thanks!